BUILD

```diff
@@ -458,6 +459,12 @@ cc_binary(
 # Tests
 ################################################################################
 
+filegroup(
+    name = "testdata",
+    visibility = ["//:__subpackages__"],
+    srcs = glob(["src/google/protobuf/testdata/**/*"]),
```

Comment:
Can we list these explicitly? My recent experiences have led me to hate `glob()`.

Reply:
+1

Reply:
glob isn't an antipattern, and given the fact that at this moment in time our story for keeping lists of explicit files in sync is sort of a mess, I'm in favor of this causing loud failures vs silent failures for the time being.

Reply:
Discussed offline, once we have file_list.bzl imported, we can refactor this.